src/app/modules/common/interface-status-icon/interface-status-icon.component.ts

```diff
@@ -39,11 +42,25 @@ export class InterfaceStatusIconComponent implements OnChanges {
 
   ngOnChanges(): void {
     if (this.update?.sent_bytes_rate > this.minRate) {
-      this.tableService.updateStateInfoIcon(this.elementId, 'sent');
+      this.updateStateInfoIcon('sent');
     }
 
     if (this.update?.received_bytes_rate > this.minRate) {
-      this.tableService.updateStateInfoIcon(this.elementId, 'received');
+      this.updateStateInfoIcon('received');
     }
   }
+
+  updateStateInfoIcon(type: 'sent' | 'received'): void {
+    if (!this.stateIcon) {
+      return;
+    }
+    const arrowIcons = this.stateIcon._elementRef.nativeElement.querySelectorAll('.arrow');
+    const targetIconEl = type === 'sent' ? arrowIcons[0] : arrowIcons[1];
+
+    targetIconEl.classList.add('active');
+
+    timer(2000).pipe(untilDestroyed(this)).subscribe(() => {
```

Comment:
I think we can add `take(1)` to ensure this one is taken only once.

Reply:
The timer operator only emits one so no need here.